stdlib/public/runtime/Paths.cpp

```diff
@@ -524,7 +534,8 @@ _swift_initRuntimePath(void *) {
 #if APPLE_OS_SYSTEM
   const char *path = dyld_image_path_containing_address(_swift_initRuntimePath);
 
-  runtimePath = ::strdup(path);
+  // No need to ::strdup() this, apparently
```

Comment:
I feel like someone is going to come by in five years and be very confused about "apparently." Suggest, "as the return value from dyld_image_path_containing_address is guaranteed to remain valid as long as the library is loaded."